app/models/topic_timer.rb

```diff
@@ -100,92 +138,86 @@ def executed_at_in_future?
     ))
   end
 
+  def publishing_to_category?
+    self.status_type.to_i == TopicTimer.types[:publish_to_category]
+  end
+
   # TODO(martin - 2021-05-01) - Remove cancels for toggle_topic_closed once topic timer revamp completed.
   def cancel_auto_close_job
     Jobs.cancel_scheduled_job(:toggle_topic_closed, topic_timer_id: id)
-    Jobs.cancel_scheduled_job(:close_topic, topic_timer_id: id)
+    Jobs.cancel_scheduled_job(TopicTimer.type_job_map[:close], topic_timer_id: id)
```

Comment:
I'm not sure if replacing `:close_topic` with `TopicTimer.type_job_map[:close]` is better here since it makes the code longer and adds a layer of symbol over another symbol.